README.md

```diff
@@ -684,6 +684,203 @@ var callbacks = {
+        var formSelector = '#payment-form';
+
+        // Jquery-style selector pointing to your verify form
+        var verifyFormSelector = '#verify-form';
+
+        BillForward.captureBankAccountOnSubmit(formSelector, 'stripe', accountID, callback);
+
+        function callback(paymentMethod, error) {
+            if (error) {
+                console.error(error);
+            } else {
+                console.log(paymentMethod);
+
+                // Do some stuff once the capture is complete
+            }
+        }
+```
+
+#### Capture directly with code
+
+Is possible to capture a bank account directly with javascript without the usage of an `html form`.
+Follows an example:
+
+```javascript
+var bankDetails = {
+    "routingNumber":"110000000",
+    "accountNumber":"000123456789",
+    "holderName":"John Doe",
+    "bankAccountName":"John Doe's Bank",
+    "accountHolderType":"company"
+};
+
+BillForward.captureBankAccount(bankDetails, 'stripe', accountID, callback);
+
+```
+
+#### Verify with a form
+
+After we have successfully created a bank account we have to verify it to being able to use it for a payment. The way `stripe` verify its bank accounts is through `micro deposits`.
+After a bank account is created, two micro deposits (of the order of a couple tens of cents) are being made on the account bank account. The two amounts have to  be used as input values for the next step.
+
+Here is how the `html` code is supposed to look like:
+
+```html
+<form id="verify-form">
+    <span class="payment-errors"></span>
+    <span class="payment-success"></span>
+
+    <div class="form-row">
+        <label>
+            <span>Payment ID</span>
+            <input type="text" size="45" bf-data="paymentMethodID" value="PAYMENT METHOD ID"/>
+        </label>
+    </div>
+
+    <div class="form-row">
+        <label>
+            <span>Amount 1</span>
+            <input type="text" size="4" bf-data="amount1" value="FIRST MICRODEPOSIT AMOUNT"/>
+        </label>
+    </div>
+
+    <div class="form-row">
+        <label>
+            <span>Amount 2</span>
+            <input type="text" size="4" bf-data="amount2" value="SECOND MICRODEPOSIT AMOUNT"/>
+        </label>
+    </div>
+
+    <button type="submit">Submit Verify</button>
+</form>
+```
+
+Follows the `javascript` code needed to bootstrap the form:
+
+ ```javascript
+
+BillForward.verifyBankAccountOnSubmit(verifyFormSelector, 'stripe', accountID, verifyCallback);
+
+ function verifyCallback(paymentMethod, error) {
+             if (error) {
+                 console.error(error);
+             } else {
+                 console.log(paymentMethod);
+                 //Do stuff...
+             }
+         }
+ ```
+
+#### Verify with code
+
+Like for the capturing it's possible to do the verification step through code only:
+
+```javascript
+var bankDetails = {
+        "amounts":["32","45"]
+    };
+
+BillForward.verifyBankAccount(bankDetails, 'stripe', accountID, callback);
```

Comment:
Does bank account verification work with public token?

I see on backend that the privileges for `POST /payment-methods/{payment-method-ID}/verify/micro-deposits` are:

```java
@PreAuthorize("@BFSecurityService.canPut(authentication, #paymentMethod, 'PaymentMethod', 'Edit')")
```

This does not match how we achieved public token auth on `POST /tokenization/ach`:

```java
@PreAuthorize("@BFSecurityService.canPost(authentication, #account, 'BFJS', 'Tokenize')")
```